src/System.Private.CoreLib/shared/System/Text/EncoderNLS.cs

```diff
@@ -38,6 +39,7 @@ internal EncoderNLS(Encoding encoding)
 
         internal EncoderNLS()
         {
+            // TODO-NULLABLE: _encoding doesn't seem to ever be null and this constructor has 0 refs
```

Comment:
Cool.  Let's just delete the ctor then. cc: @tarekgh

Reply:
@stephentoub one of the classes which had similar constructor (DecoderNLS I think) also mentioned something about being used by deserialization so I decided to leave it just in case for now

Reply:
> also mentioned something about being used by deserialization

We don't support serialization of encodings/encoders/decoders in .NET Core, do we?
cc: @ViktorHofer

Reply:
Looking at the IL generated code look we already eliminate this constructor. I am not seeing it in the IL :-)

---
In reply to: [272194140](https://github.com/dotnet/coreclr/pull/23708#discussion_r272194140) [](ancestors = 272194140)

Reply:
If such types are not part of this list then not: https://docs.microsoft.com/en-us/dotnet/standard/serialization/binary-serialization#binary-serialization-in-net-core

Reply:
removed, thanks for clarifying